models/message.js

```diff
@@ -0,0 +1,61 @@
+const { Schema, model } = require('mongoose')
+const {
+  FIELD_CANNOT_BE_EMPTY,
+  ENUM_CAN_BE_ONE_OF,
+  FIELD_CANNOT_BE_LONGER,
+  FIELD_CANNOT_BE_SHORTER
+} = require('~/consts/errors')
+const { USER, MESSAGE, CHAT } = require('~/consts/models')
+const {
+  enums: { AUTHOR_ROLE_ENUM }
+} = require('~/consts/validation')
+
+const messageSchema = new Schema(
+  {
+    author: {
+      type: Schema.Types.ObjectId,
+      ref: USER,
+      required: [true, FIELD_CANNOT_BE_EMPTY('author')]
+    },
+
+    authorRole: {
+      type: String,
+      enum: {
+        values: AUTHOR_ROLE_ENUM,
+        message: ENUM_CAN_BE_ONE_OF('initiator role', AUTHOR_ROLE_ENUM),
+        required: [true, 'Author role must be selected.']
+      }
+    },
+
+    messageText: {
```

Comment:
just text will be more concise (in my opinion)

Reply:
I agree